### DISCUSSION TITLE
where is header styling?

### DISCUSSION THREAD

**Discussion by robertIsaac** [vmware-clarity/core#65] [Q&A]
now I'm trying [App Layout](https://clarity.design/foundation/app-layout/)
but this code is not working
for example what the class `header-6` is part of?
it's not in `@cds/city/css/bundles/default.min.css` or in `@cds/core/global.min.css` which is what said need to be imported in the [Developing](https://clarity.design/get-started/developing/) section

**Comment by ashleyryan** [comment]
That website covers a bit of the older Clarity Angular (and @clr/ui, the older html/css implementation) and the newer Clarity Core `@cds/core` versions of the design system.  We don't yet have a header component at this time in Clarity Core.

If you visit the angular version of the site, you can find the Header component there, which does have an html/css-only implementation (if you're not using Angular): https://angular.clarity.design/documentation/header
